scripts/templates/create-component/EmptyState.mustache

```diff
@@ -1,20 +1,20 @@
-import { I{{componentName}}Props, I{{componentName}}ViewProps } from './{{componentName}}.types';
-import { BaseState } from '../../utilities/BaseState';
+import { useState } from 'react';
+import { I{{componentName}}Component, I{{componentName}}Props, I{{componentName}}ViewProps } from './{{componentName}}.types';
+import { useControlledState } from '../../Foundation';
```

Comment:
Should this use `@uifabric/foundation` instead?

Reply:
I don't thinks so. These are the comment entry points that our components use in OUFR and experiments.